zerver/openapi/zulip.yaml

```diff
@@ -9142,6 +9142,47 @@ paths:
       responses:
         "200":
           $ref: "#/components/responses/SimpleSuccess"
+  /mobile_push/test_notification:
+    post:
+      operationId: test-notify
+      summary: Sends a test notifications to the user's mobile device(s)
+      tags: ["mobile"]
+      description: |
+        This endpoint allows a user to trigger a test push notification to their
+        selected mobile devices, or all their mobile devices.
```

Comment:
s/devices/device/ (for the first occurrence) — you can select one, or let it go to all, but not select multiple